drivers/adc/adc_nrfx_saadc.c

```diff
@@ -544,6 +605,8 @@ static const struct adc_driver_api adc_nrfx_driver_api = {
 #endif
 #if defined(CONFIG_SOC_NRF54L15)
 	.ref_internal  = 900,
+#elif defined(CONFIG_SOC_NRF54H20)
+	.ref_internal  = 1024,
```

Comment:
Looks like something that could be moved to DT in future.

Reply:
You are right.